app/tests/test_dependencies.py

```diff
@@ -245,32 +203,19 @@ async def make_call(route_suffix: str = "", headers=None):
     await make_call(
         route_suffix="/admin",
         headers={
-            "x-api-key": "ecosystem-admin.provided-api-key",
+            "x-api-key": "tenant-admin.provided-api-key",
         },
     )
     # then
-    assert injected_controller.base_url == Role.ECOSYSTEM.agent_type.base_url
+    assert injected_controller.base_url == Role.TENANT.agent_type.base_url
     assert injected_controller.client.headers["x-api-key"] == "provided-api-key"
     assert isinstance(injected_controller, AcaPyClient)
 
-    # when
-    response = await make_call(
-        route_suffix="/admin",
-        headers={
-            "x-api-key": "member-admin.provided-x-api-key-1",
-        },
-    )
-    # then
-    assert response.status_code == 200
-    assert injected_controller.base_url == Role.MEMBER.agent_type.base_url
-    assert injected_controller.client.headers["x-api-key"] == "provided-x-api-key-1"
-    assert isinstance(injected_controller, AcaPyClient)
-
 
 @pytest.mark.asyncio
-async def test_ecosystem_admin_agent():
-    async with asynccontextmanager(dependencies.agent_role(role=Role.ECOSYSTEM_ADMIN))(
-        auth=AcaPyAuth(role=Role.ECOSYSTEM_ADMIN, token=GOVERNANCE_ACAPY_API_KEY)
+async def test_tenant_admin_agent():
```

Comment:
This test/a test with this name already exists in line 43. Looks duplicate to me. Remove one if so or change the logic to what you wanna test

Reply:
have you removed this duplicate?